### DISCUSSION TITLE
Request Introspection Query recieve scheme GraphQl

### DISCUSSION THREAD

**Discussion by NeutronBro** [requarks/wiki#6422] [Error / Bug Report] [answered]
On a request Introspection Query in API web application, return scheme GraphQl which show all architecture of application. 
Closing API - closing process of authentication.

**Comment by NGPixel** [comment, marked as answer]
Wiki.js is open source and that schema is already public to anyone who [looks up the github repository](https://github.com/requarks/wiki/tree/main/server/graph/schemas)... Disabling introspection is pointless in this case.

You can disable introspection by setting the environment variable `NODE_ENV` to `production`
